charts/pro-api/values.yaml

```
@@ -377,7 +381,7 @@ permissionsPodSettings:
       memory: 256M
```

Давайте тогда запрос тоже поправим, чтобы нам хватало ресурсов на ноде при деплое.
```suggestion
      memory: 512M
```

---

добавил